packages/models/notification.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import Avatar from '../UI/src/components/atoms/Avatar/Avatar';
+
+export type NotificationUser = {
+  data: {
+    id: number;
+    attributes: {
+      username?: string;
```

Comment:
@yacineKahlerras is there any specific reason we are marking this as optional?

Reply:
@dbradham yeah for some of the accounts they ca be null, so i left it as an option to maybe put an email address if the field doesn't exist  do you want me to set the type as `username: string | null` instead ?